### DISCUSSION TITLE
editor settings per project ?

### DISCUSSION THREAD

**Discussion by fmieres** [alefragnani/vscode-project-manager#537] [Q&A] [answered]
Is there a way to set editor settings per project ? Like, for instance, `tabSize` setting. So project A has this set as 2 and project B as 4

**Comment by alefragnani** [comment, marked as answer]
Hi @fmieres ,

This is not the extension's idea. And in fact, you can do that with VS Code, natively. You just have to define those settings for each `Workspace` you open, which may be a simple folder or a multi-root folder scenario (which can cave a setting applied to all contained folders).

More info on VS Code docs:
* https://code.visualstudio.com/docs/getstarted/settings
* https://code.visualstudio.com/docs/editor/multi-root-workspaces

Hope this helps
